packages/rocketchat-lib/server/methods/sendMessage.coffee

```diff
@@ -49,8 +49,9 @@ Meteor.methods
 		message.alias = user.name if not message.alias? and RocketChat.settings.get 'Message_SetNameToAliasEnabled'
 		if Meteor.settings.public.sandstorm
 			message.sandstormSessionId = this.connection.sandstormSessionId()
-
+		
 		RocketChat.sendMessage user, message, room
+		RocketChat.metrics.messagesSent.inc()
```

Comment:
This line broke several things which rely on the return result of `RocketChat.sendMessage`.